advanced_java/enum/MyClass4.java

```diff
@@ -0,0 +1,29 @@
+public class MyClass4 {
+  public static void main(String[] args) {
+    Day myVar = Day.WEDNESDAY;
+
+    switch(myVar) {
+      case MONDAY:
+        System.out.println("Workday.");
+        break;
+      case TUESDAY:
+         System.out.println("Presentaion day.");
+        break;
+      case WEDNESDAY:
+        System.out.println("Workday.");
+        break;
+      case THURSDAY:
+        System.out.println("Workday.");
+        break;
+      case FRIDAY:
+        System.out.println("PARTY!");
+        break;
+      case SATURDAY:
+        System.out.println("Relax.");
+        break;
+      case SUNDAY:
+        System.out.println("Complete deadlines.");
+        break;
```

Comment:
Move all the constants to a final variable and use it instead of hardcoding them in here.

Reply:
DO you mean the statements I am displaying?